test/NuGet.Clients.Tests/NuGet.SolutionRestoreManager.Test/VsSolutionRestoreServiceTests.cs

```diff
@@ -2110,6 +2111,39 @@ public void ToPackageSpec_TargetFrameworkWithAlias_DefinesAliasCorrectly()
             Assert.Equal("tfm2", targetFrameworkInfo.TargetAlias);
         }
 
+        // In VS, when a PackageReference CPS (SDK style) project gets loaded, the project system needs to send us a nomination.
```

Comment:
I think this comment is very good at explaining this. Do we need to create/amend some developer docs (*cough* blog) somewhere to explain it in equal or more detail?